src/ExEmElm/Types.elm

```diff
@@ -85,6 +88,36 @@ attribute name value =
         }
 
 
+childrenOfNode : Node -> List Node
+childrenOfNode node =
+    case node of
+        Element tag attrs children ->
+            children
+
+        _ ->
+            []
+
+
+tagOfNode : Node -> String
+tagOfNode node =
+    case node of
+        Element tag attrs children ->
+            tag
+
+        _ ->
+            ""
```

Comment:
Not sure about the use of `""` as a sentinel. Have you considered using `Maybe String` as a return type?